src/translations/en.js

```diff
@@ -605,7 +605,7 @@ export default {
 		"invalidLink_msg": "Sorry, this link is not valid.",
 		"invalidMailAddress_msg": "The following email addresses could not be invited because they are invalid:",
 		"invalidPassword_msg": "Invalid password. Please check it again.",
-		"invalidRecipients_msg": "Please correct the invalid email address(es) in the recipient fields.",
+		"invalidRecipients_msg": "Please correct or remove the invalid email addresses:", // TODO
```

Comment:
Still a `TODO` here

Reply:
I added and changed the keys in phrase now.